### DISCUSSION TITLE
Is easy effects software supported?

### DISCUSSION THREAD

**Discussion by Josh65-2201** [MichaIng/DietPi#6678] [Q&A] [answered]
https://github.com/wwmm/easyeffects

**Comment by Joulinar** [comment, marked as answer]
Should be as there is an official Debian package available that could be installed using `apt` package manager https://packages.debian.org/search?suite=all&searchon=names&keywords=easyeffects

**Reply by MichaIng** [reply]
Hence:
```sh
apt install easyeffects
```
